### PR TITLE
Improvements to Manage Newsroom > Smart Contract roles

### PR DESCRIPTION
**Pull request by tobek**
While debugging and testing the add-new-newsroom-admin flow I came across several bugs and missing features that would be helpful both for users and for us debugging, see commit messages for details.

### CODE REVIEW

File: packages/components/src/Contributors/ContributorCount.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class ContributorCount extends React.Component<ContributorsCountProps> {
           {this.props.displayedContributors.slice(0, 3).map((contributor: any, i: number) => {
             return (
               <ContributorCountAvatar key={i}>
-                {contributor.payerChannel.tiny72AvatarDataUrl ? (
+                {contributor.payerChannel && contributor.payerChannel.tiny72AvatarDataUrl ? (
```
Comment (nickreynolds) [inline packages/components/src/Contributors/ContributorCount.tsx]: might be a conflict on this line cause I already fixed this in another PR, didn't I?

Comment (tobek) [reply packages/components/src/Contributors/ContributorCount.tsx]: You sure did - my commit disappeared upon rebasing.

**Comment by dankins** [comment]
:tada: This PR is included in version 4.10.0 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**